### DISCUSSION TITLE
Integration Mod for Conan Exiles

### DISCUSSION THREAD

**Discussion by Sarpendon** [gameserverapp/Platform#245] [Suggestions]
Hey Max, was thinking about maybe making an integration mod similar to the one for Ark you provide, I been modding for Conan now for 2 years and I think this would help a lot regarding the economy of tokens and more to combine your awesome work with a mod. Let me know what you think. Best wishes.

**Comment by Krysez** [comment]
There have been some update since the suggestion and wondered if the idea had been revisited?

**Reply by mvaessen** [reply]
Hi, I generally follow the number of upvotes to determine what will be worked on next. As of right now the number of votes for this suggestion wont put it in the top 5 (most upvoted suggestions).

In the coming months I will be working on suggestions and game requests that received the most upvotes. This will most likely be done using polls for paying customers only.

Edit: A Conan modder offered me their help some time ago, which I accepted under the condition they were able to provide examples that GSA dev team could evaluate to determine level of skills. I haven't received any info as of yet. So another reason why nothing happened is because we have no Conan mod dev.

**Comment by camerapup** [comment]
upvoting this as well cause we recently added a conan, and palworld map to our gsa dashboard
